src/Schema/Values/OwnerType.ts

```diff
@@ -6,6 +6,8 @@
 export enum OwnerType {
   activities = "activities",
   allArtistSeries = "allArtistSeries",
+  alertConfirmation = "alertConfirmation",
+  alertFilters = "alertFilters",
```

Comment:
**suggestion**: I don't think we need these values, which map to the specific steps in the alert flow. I think those are better captures as ContextModule values.

Although we might consider additions to eventually replace the existing `savedSearch` and `savedSearches` values such as `alert` and `alerts`.

Reply:
Does that mean that you suggest excluding the owner_type from the event at all and use ContextModule instead? Wha value would you suggest to use for the ContextModule in this case?
[dock with the event details](https://docs.google.com/spreadsheets/d/1iBmzvJwgZ21J8w2VbVY3JAbH0EHbvUMz77HnZs3Ht9U/edit#gid=62721100)

Reply:
Ahh, you're totally right. For the app, we need these new owner types as there are distinct screens to track. In that case don't forget to include the in the `ScreenOwnerType` values below in this same file. While you're in here, maybe adding `alerts` and `alertDetails` as additional `ScreenOwnerType` values?